org.flozz.yoga-image-optimizer.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally it should be maintained in your GitHub repo, not here.

Reply:
Yes I will move this file to the upstream project :)